### PR TITLE
Create animals table

### PR DESCRIPTION
**Pull request by emmyobonyo**
### Tasks Completed.
- Create `vet_clinic` database.
- Created `animals` table.
- Created queries to insert data into the table.
- Created queries to query information from the table.

### CODE REVIEW

File: queries.sql [hunk 1]
```diff
@@ -1,3 +1,10 @@
 /*Queries that provide answers to the questions from all projects.*/
 
-SELECT * from animals WHERE name = 'Luna';
+SELECT * FROM animals WHERE name LIKE '%mon%';
```
Comment (Emmyn5600) [inline queries.sql:3]: - [ ] Using %mon%, you are trying to find any values that have "mon" in any position, however, we are required to find all animals that its name ends in "mon".

File: queries.sql [hunk 2]
```diff
@@ -1,3 +1,10 @@
...
+SELECT * FROM animals WHERE date_of_birth BETWEEN 2016 AND 2019;
```
Comment (Emmyn5600) [inline queries.sql:4]: - [ ] If we decided to use this way, let us use the `EXTRACT` function.

**Review by Emmyn5600 — CHANGES_REQUESTED** [review]
Hi, @emmyobonyo 

Good job so far! 👏🏼 

Great work on making the project requirements:clap:
You've done well implementing the project requirements, but there are still some issues which aren't been addressed yet 
 
## STATUS: CHANGES REQUESTED :recycle:


### Changes requested.

- [ ] Check the comment under the review.

 #### Optional suggestions
 
 - N/A
 
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

#### _**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**_
------------------------------------------------------------------------------------------------------------------------------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)

File: queries.sql (hunk 1 above)
Comment (emmyobonyo) [reply queries.sql:3]: Done 👍

File: queries.sql (hunk 2 above)
Comment (emmyobonyo) [reply queries.sql:4]: Done 💯

**Review by Emmyn5600 — APPROVED** [review]
Hi @emmyobonyo 

Good job👏🏼

## STATUS: APPROVED :muscle::clap:

Congratulations! :tada:
Your project is complete!  It's time to merge it :shipit:
Great work on implementing the changes requested by the previous reviewer.

 #### Optional suggestions
 -  N/A.
 
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

#### _**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**_
------------------------------------------------------------------------------------------------------------------------------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)

**Pull request merged**